mlflow/server/js/src/components/ExperimentListView.js

```diff
@@ -57,19 +118,36 @@ export class ExperimentListView extends Component {
                 className = `${className} active-experiment-list-item`;
               }
               return (
-                <Link
-                  style={{ textDecoration: 'none', color: 'unset' }}
-                  key={e.getExperimentId()}
-                  to={Routes.getExperimentPageRoute(e.getExperimentId())}
-                  onClick={active ? ev => ev.preventDefault() : ev => ev}
-                >
-                  <div
-                    className={className}
-                    title={e.getName()}
+                <div key={e.getExperimentId()} title={e.getName()}
+                  className={`header-container ${className}`}>
+                  <Link
+                    style={{ textDecoration: 'none', color: 'unset', width: '80%'}}
+                    to={Routes.getExperimentPageRoute(e.getExperimentId())}
+                    onClick={active ? ev => ev.preventDefault() : ev => ev}
```

Comment:
I generally agree but figured that the following problem had to be solved:
If a user is e.g. reviewing experiment `2`, the URL will point to `/experiments/2`. After deleting the experiment the user would be redirected to the "Something went wrong..." screen since the UI is not handling the display of experiments which are associated with the lifecycle state "Deleted".

In the beginning, I actually looked into different solutions for the issue but I did not want to make too many assumptions about the expected behavior without getting feedback. I think the following solutions make sense and are quite easy to implement:
1. Make use of an imperative API (`history.push()`) after the deletion. This would require to wrap another component into the `withRouter` HOC.
2. Modifying the following part of the `HomeView` component and generally checking whether the experiment is active, followed by possible rerouting. However, this would change the general behavior of the application.
```js
  if (ownProps.experimentId === undefined) {
    const firstExp = getFirstActiveExperiment(getExperiments(state));
    if (firstExp) {
      return { experimentId: parseInt(firstExp.experiment_id, 10) };
    }
  }
  return {};
```

If you could provide me with the expected behavior I could change it accordingly.

I tried to mention that in the description of the PR:

> The "Delete Experiment" option is disabled for the currently active/selected experiment. Alternatively, the expected behavior needs to be defined for this case.